src/migrations/migrations-truffle-5/4_deploy_Pool.js

```diff
@@ -3,10 +3,18 @@
 abi = require("ethereumjs-abi")
 
 console.log("TRADING PERIOD DAYS ENV? ", process.env.TRADING_PERIOD_DAYS)
-const TRADING_PERIOD_IN_HOURS = (process.env.TRADING_PERIOD_DAYS || 3) * 60 * 60 * 24 // 3 days for testing
+console.log("TRADING_END_TIME ENV? ", process.env.TRADING_END_TIME)
+
+var TRADING_PERIOD_IN_HOURS 
+if (process.env.TRADING_END_TIME) {
+  tradingEndTime = new Date(Date.parse(process.env.TRADING_END_TIME))
+  currentTime = new Date()
+  TRADING_PERIOD_IN_HOURS = Math.floor(Math.abs(tradingEndTime - currentTime) / 36e5)
+} else {
+  TRADING_PERIOD_IN_HOURS = (process.env.TRADING_PERIOD_DAYS || 3) * 60 * 60 * 24 // 3 days for testing
```

Comment:
Imo, is better to throw an exception here, instead of using this other method. I guess is for the tests, but in that case, I would make the test to define `TRADING_END_TIME` that expires in 3 dates. 

This way, on every deployment it becomes mandatory to provide `TRADING_END_TIME`